SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -329,6 +337,12 @@ void USpatialReceiver::OnRemoveComponent(const Worker_RemoveComponentOp& Op)
 		return;
 	}
 
+	// UNR-3066 Remove all queued owner only components for which we haven't received ownership yet
+	PendingOwnerOnlyComponents.RemoveAll([&Op](const PendingAddComponentWrapper& Queued)
```

Comment:
Don't we only need to do this if Op.component_id is of type SCHEMA_OwnerOnly ?

Reply:
`PendingOwnerOnlyComponts` only contains components of that type.

Reply:
I meant only check if we need to remove if the component that's being removed is of type SCHEMA_OwnerOnly

Reply:
Good catch